modules/logging.py

```diff
@@ -64,9 +67,32 @@ async def logging_loop(self) -> None:
 
         embed = to_log.__dict__.get("embed") or discord.utils.MISSING
 
+        avatar_url: str | None = core.CONFIG["LOGGING"].get("webhook_avatar_url")
+        avatar_name = "PythonistaBot Logging"
+
+        if not avatar_url and self.user is not None and "runner" in core.CONFIG["LOGGING"]:
+            runner_id: int = core.CONFIG["LOGGING"]["runner"]
+            user = self.user or self.bot.get_user(runner_id)
+
+            if user:
+                avatar_url = user.display_avatar.url
+                avatar_name = f"Logging: Dev: {user.display_name}"
+
+            else:
+                try:
+                    user = await self.bot.fetch_user(
```

Comment:
You can *now* use `Bot.get_or_fetch_user` to avoid the absolute API call here. It's a single guild bot so we don't worry about chunking!